client/state/data-layer/wpcom/gravatar-upload/test/index.js

```diff
@@ -54,10 +54,6 @@ describe( '#announceSuccess()', () => {
 			},
 		} ) );
 	} );
-	afterAll( () => {
```

Comment:
Do we still need to restore the previous FormData implementation?

Reply:
I don't think we need it; just removed it.